docs/modules/pipelines/pages/configuring-jobs.adoc

```diff
@@ -169,3 +174,26 @@ cluster change depending on these two settings.
 |suspend job
 |===
 
+== Configuring a User Code Namespace
+
+Normal Jet jobs can be configured to use a xref:clusters:user-code-namespaces.adoc[User Code Namespace] (UCN), which provides them
+access to custom resources associated with that UCN. {ucn} must be enabled and a namespace configured with the desired resources
+created before it can be used in jobs. Any data structures used in these jobs should also have their UCN configured to ensure
+operations for that data structure also have access to the necessary custom resources. Light jobs do not support UCN.
+
+{ucn} are managed separately to the Jet system, so these resources persist between individual jobs. This support allows for jobs to
+be submitted and updated while still being able to use the same custom resources within pipelines.
+
+WARNING: {ucn} can be dynamically updated at runtime, but it is recommended that namespaces associated with Jet jobs are not changed
+while those jobs are running. Doing so may cause job execution inconsistency between member nodes if jobs restart during the namespace update. If all related jobs are suspended, then the associated namespace can be safely updated before resuming jobs.
+
+The following is an example of how to associate a namespace with a job using the Java API. The namespace must already be configured
+in the Hazelcast member configuration.
+
+[source,java]
+----
+JobConfig jobConfig = new JobConfig();
+jobConfig.setUserCodeNamespace("my_namespace");
+----
+
+NOTE: SQL jobs do not support user defined functions and so definition of a UCN is not supported via SQL.
```

Comment:
you might add a note about unsupported parameter in https://deploy-preview-1383--hardcore-allen-f5257d.netlify.app/hazelcast/6.0-snapshot/sql/create-job#parameters

Reply:
Good shout, thanks: https://github.com/hazelcast/hz-docs/pull/1383/commits/e692a00448e05d8c6164ee2feea6fd83f7299108

Reply:
it should be more readable inline in the table, similar to https://deploy-preview-1383--hardcore-allen-f5257d.netlify.app/hazelcast/6.0-snapshot/sql/alter-job

Reply:
I've reverted my addition to `create-job.adoc`, I don't think we need to mention it actually based on the alter job documentation; both pages state "the following parameters are supported", so the documentation not including `userCodeNamespace` already implies it is not supported, without cluttering the page.

Reply:
`CREATE JOB` doc links only to list of parameters in Jet without mentioning than UCN is not supported. It is reference, so it should be complete.

Reply:
I was totally missing this part when scanning over the page - my apologies. I think I've got it this time: https://github.com/hazelcast/hz-docs/pull/1383/commits/f4c08e40f6d41fb0fe7bc21cbc5c1f833ef8d766